### DISCUSSION TITLE
Swagger UI not rendering /api/workflow/{workflowId}

### DISCUSSION THREAD

**Discussion by maheshyaddanapudi** [Netflix/conductor#2192] [Q&A] [answered]
Hi,

I am using the Conductor Version: 3.1.0 and in the OpenAPI UI / Swagger UI, the URL /api/workflow/{workflowId} is not rendered by Swagger UI and timesout after a while making the page unresponsive. However, if I directly access /api/workflow/6f5959c8-9849-4cf9-b630-c476b83a29f7 or something similar with a workflowId in browser window, it gives the expected response.

Unsure if this has anything to do with core functionality than just Swagger documentation.

P.S. I have compared the Swagger definition for similar methods / APIs like /api/metadata/workflow/{name} which also takes in a single String parameter (optionally version as second parameter) and the Swagger definitions looked similar accordingly. Couldn't figure out why this alone is not rendering.

Also tried changing the version from 1.4+ to 1.2.32 which was working for me on a different Spring Boot project (For Reference : https://github.com/maheshyaddanapudi/APICallsScheduler)

![Screenshot 2021-04-20 at 2 01 47 PM](https://user-images.githubusercontent.com/24351133/115365073-a998a980-a1e1-11eb-8e05-ff5888188c83.png)

Thanks and Regards, 
Mahesh Yaddanapudi.

**Comment by kishorebanala** [comment]
https://github.com/Netflix/conductor/issues/2174#issuecomment-827218539

**Comment by apanicker-nflx** [comment, marked as answer]
Should be fixed as part of #2210
